dev/com.ibm.ws.microprofile.reactive.streams.operators.cdi/bnd.bnd

```diff
@@ -0,0 +1,73 @@
+#*******************************************************************************
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-Name: com.ibm.ws.microprofile.reactive.streams.operators.cdi
+Bundle-SymbolicName: com.ibm.ws.microprofile.reactive.streams.operators.cdi
+
+Export-Package: \
+  !com.ibm.ws.microprofile.reactive.streams.cdi.resources, \
+  com.ibm.ws.microprofile.reactive.streams.cdi*
+    
+Import-Package: \
+  javax.enterprise.event;version="[1.1,2.1)",\
```

Comment:
perhaps should be `javax.enterprise.*; version="[1.1,3)",\`

Reply:
Done.